### PR TITLE
robot_control: correct read(..)/write(..) overrides.

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
I don't believe the CI failures are related to db8e6eb:

```python-traceback
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apt/cache.py", line 255, in __getitem__
    return self._weakref[key]
  File "/usr/lib/python3.5/weakref.py", line 131, in __getitem__
    o = self.data[key]()
KeyError: 'ros-kinetic-gazebo-ros'
```

File: khi_robot_control/include/khi_robot_hardware_interface.h [hunk 1]
```diff
@@ -56,8 +56,8 @@ class KhiRobotHardwareInterface : public hardware_interface::RobotHW
     bool hold();
     void deactivate();
     void close();
-    void read( const ros::Time time, const ros::Duration period );
-    void write( const ros::Time time, const ros::Duration period );
+    void read( const ros::Time& time, const ros::Duration& period );
+    void write( const ros::Time& time, const ros::Duration& period );
```
Comment (gavanderhoorn) [inline khi_robot_control/include/khi_robot_hardware_interface.h]: Suggestion: add `override` keyword here.

Comment (d-nakamichi) [reply khi_robot_control/include/khi_robot_hardware_interface.h]: Yes, it is better to add.

Comment (gavanderhoorn) [reply khi_robot_control/include/khi_robot_hardware_interface.h]: @d-nakamichi: would you want me to add it in this PR?

Comment (d-nakamichi) [reply khi_robot_control/include/khi_robot_hardware_interface.h]: Yes I want, because adding `override` keyword is closely related to this PR.

Comment (gavanderhoorn) [reply khi_robot_control/include/khi_robot_hardware_interface.h]: Done in bb6dc4d.

**Comment by d-nakamichi** [comment]
@gavanderhoorn Thank you, I merged.
Travis is now failing, but it is not due to this. (Try to fix by #35)

**Pull request merged**